app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointment.kt

```diff
@@ -134,5 +135,44 @@ data class OverdueAppointment(
         facilityUuid: UUID,
         scheduledBefore: LocalDate
     ): Observable<List<OverdueAppointment>>
+
+    @Query("""
+        SELECT * FROM (
+          $OVERDUE_APPOINTMENTS_QUERY 
+             JOIN PatientFts patientFts ON patientFts.uuid = A.patientUuid
+             WHERE
+               IFNULL(patientAssignedFacilityUuid, appt_facilityUuid) = :facilityUuid AND
+               appt_scheduledDate < :scheduledBefore AND
+               P.deletedAt IS NULL AND
+               P.status != 'dead' AND 
+               PPN.deletedAt IS NULL AND 
+               A.deletedAt IS NULL AND 
+               (A.status = 'scheduled' OR A.status  = 'cancelled') AND 
+               (BP.recordedAt IS NOT NULL OR BloodSugar.recordedAt IS NOT NULL) AND
```

Comment:
Any reason we didn't pull out the common conditions to outside query? Because we are duplicating them in both search sub-queries.